.github/workflows/stalebot.yml

```diff
@@ -0,0 +1,25 @@
+name: Close stale Issues and PRs
+
+permissions:
+  issues: write
+  pull-requests: write
+
+on:
+  schedule:
+  - cron: "00 0 * * *" # runs at 00:00 daily
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v9.0.0
+      name: Clean up stale PRs and Issues
+      with:
+        stale-pr-message: "👋 This pull request has been marked as stale because it has been open with no activity. You can: comment on the issue or remove the stale label to hold stale off for a while, add the `Keep` label to hold stale off permanently, or do nothing. If you do nothing, this pull request will be closed eventually by the stale bot. Please see CONTRIBUTING.md for more policy details."
+        stale-pr-label: "Stale"
+        exempt-pr-labels: "Keep" # a "Keep" label will keep the PR from being closed as stale
+        days-before-pr-stale: 730 # when the PR is considered stale
+        days-before-pr-close: 60 # when the PR is closed by the bot, 
```

Comment:
should we reduce this to maybe 15 days?

Reply:
to close after getting the `stale` label, you mean?  I'm fine to do that, but was giving some more buffer to ease into the change.

Reply:
OK with your decision